services/payments/src/simcore_service_payments/services/payments_methods.py

```diff
@@ -148,14 +148,14 @@ async def create_payment_method(
     )
 
 
-async def list_payments_methods(
+async def list_successful_payment_methods(
```

Comment:
hehe, we go on and off with the names. I had these names initially and change them because at this interface I wanted to remove the concept of "state". 

A payment-method in this interface "exists" when it is fully created. the state of creation is something delegated to the repo interface but not here. 

With payments is different because you accept, cancel or reject a payment while with payment-methods we only intend to store them.

Reply:
I'm also confused by this successful in the name. I don't get it how a payment method can be successful.

Reply:
Ok, I have reverted the naming back :) The first time I was working with it, I thought it would be better as I was surprised that in the Database layer, we filter this state and I do not know about it from the higher interface. But I guess when we want to remove the concept of "state" than it doesn't matter.